stdlib/universe/universe.flux

```diff
@@ -1,6 +1,7 @@
 package universe
 
 import "system"
+import "date"
```

Comment:
this is great. Makes me think why don't we `import "math"` but we add it programmatically to the prelude @aanthony1243 .

Reply:
never thought about that.  I'm going to test it